src/jquery.uls.core.js

```diff
@@ -197,18 +205,49 @@
 			this.$resultsView.show();
 		},
 
+		createLanguageFilter: function () {
+			var lcd, languagesCount,
+				columnsOptions = {
+					wide: 4,
+					medium: 2,
+					narrow: 1
+				};
+
+			languagesCount = Object.keys( this.options.languages ).length;
+			lcd = this.$resultsView.lcd( {
+				languages: this.languages,
+				columns: columnsOptions[ this.menuWidth ],
+
+				quickList: languagesCount > 12 ? this.options.quickList : [],
+				clickhandler: $.proxy( this.select, this ),
+				source: this.$languageFilter,
+				showRegions: this.options.showRegions,
+				languageDecorator: this.options.languageDecorator
```

Comment:
Missing noResultsTemplate option for lcd

Reply:
Merged master and added `noResultsTemplate` in 90d4808.